lib/workers/repository/updates/generate.ts

```diff
@@ -293,7 +295,7 @@ export function generateBranchConfig(
     if (config.upgrades[0].depName?.startsWith('@types/')) {
       logger.debug('Found @types - reversing upgrades to use depName in PR');
       sortTypesGroup(config.upgrades);
-      config.upgrades[0].recreateClosed = false;
+      config.upgrades[0].recreateWhen = 'auto';
```

Comment:
```suggestion
      config.upgrades[0].recreateClosed = false;
```

Reply:
We should keep both right? Since we use `recreateWhen` to decide which texts to put in the PR, or should we use `reacreateClosed` there as well.

Reply:
Do we need `recreateWhen` for the PR? Or could we use the value of `recreateClosed`?

Reply:
We could use the `recreateClosed`.